### ISSUE TITLE
More Currency?

### ISSUE THREAD

**Issue by leonuts** [pranavp10/invoice-generator#1]
I love the invoice app, but after filling out the information, I just realized there's no Indonesian Rupiah (IDR). Are you planning to add?

Thank you

**Comment by pranavp10** [comment]
Yes, we're planning to add support for Indonesian Rupiah (IDR) in the near future. Thank you for bringing this to our attention! We're constantly working to improve our app and make it more accessible to users around the world. Your feedback is valuable to us.

**Comment by pranavp10** [comment]
I've added the Indonesian Rupiah (IDR) currency option, and it's now live on the website. Please take a look at the screenshot below for reference. If you encounter any further issues or have additional suggestions, feel free to let me know. Thank you @leonuts for bringing this to attention!
 
<img width="1728" alt="image" src="https://github.com/pranavp10/invoice-generator/assets/49075129/b511f74b-794b-4527-8a92-c5717c27220c">

**Comment by pranavp10** [comment]
https://github.com/pranavp10/invoice-generator/commit/7b49512e472c01201935abe8d92ad64011ea3154

**Comment by leonuts** [comment]
Thank you for the swift response 🙏
